ivy/functional/frontends/jax/devicearray.py

```diff
@@ -138,6 +138,41 @@ def sort(self, axis=-1, order=None):
             order=order,
         )
 
+    def sum(
```

Comment:
The promotion here will be done with the same approach as the one mentioned on the `prod` PR.